gobblin-config-management/src/main/java/gobblin/config/configstore/impl/SimpleVersionFinder.java

```diff
@@ -0,0 +1,80 @@
+package gobblin.config.configstore.impl;
+
+import java.util.Collection;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import gobblin.config.configstore.VersionFinder;
+
+
+public class SimpleVersionFinder implements VersionFinder<String> {
```

Comment:
I don't think we should care about the specific version string format. I suggest that we say that we have a "versions" subdirectory and every non-hidden file under it is a version. 

I don't know what our versioning scheme will be.